locale/de/LC_MESSAGES/django.po

```diff
@@ -174,55 +183,33 @@ msgstr "Interner Serverfehler."
 #: digitalstrategie/templates/base.html:8
 #: digitalstrategie/templates/includes/header.html:19
 msgid "Digital strategy"
-msgstr "Digital Strategie"
+msgstr "Digitalstrategie"
 
 #: digitalstrategie/templates/base_errors.html:4
 msgid "Error"
 msgstr "Fehler"
 
-#: digitalstrategie/templates/base_errors.html:11
+#: digitalstrategie/templates/base_errors.html:10
 msgid "Back to landing page"
 msgstr "Zurück zur Startseite"
 
 #: digitalstrategie/templates/email_base.html:19
 msgid "Hello"
 msgstr "Hallo"
 
-#: digitalstrategie/templates/includes/footer.html:9
+#: digitalstrategie/templates/includes/footer.html:8
 msgid "To the top of the page"
 msgstr "Zum Seitenanfang"
 
-#: digitalstrategie/templates/includes/footer.html:35
 #: digitalstrategie/templates/includes/header.html:9
 msgid "To the home page of Berlin.de"
 msgstr "Zur Startseite von Berlin.de"
 
-#: digitalstrategie/templates/includes/footer.html:62
-#, fuzzy
-msgid ""
-"Berlin.de is a service of the State of Berlin and BerlinOnline Stadtportal "
-"GmbH & Co. KG."
-msgstr ""
-"Berlin.de ist ein Angebot des Landes Berlin und der BerlinOnline Stadtportal "
-"GmbH & Co. KG."
-
-#: digitalstrategie/templates/includes/footer.html:63
-#, fuzzy, python-format
-msgid ""
-"For more information, please see the <a href=\"%(imprint_url)s\">imprint</a>."
-msgstr ""
-"Weitere Informationen hierzu finden Sie im <a href=\"%(imprint_url)s"
-"\">Impressum</a>."
-
 #: digitalstrategie/templates/includes/header.html:18
 #: digitalstrategie/templates/includes/header.html:19
 msgid "To the home page"
 msgstr "Zur Startseite"
 
-#: digitalstrategie/templates/includes/header.html:19
-msgid "Digital strategy"
-msgstr "Digitalstrategie"
-
```

Comment:
@fuzzylogic2000 I just deleted this but only from the german file, no clue why it couldn't just do it's usual thing, that cool?